src/Card.tsx

```diff
@@ -37,13 +42,14 @@ export default class Card {
         this.name = name;
         this.numInstances = num_instances;
         this.cardType = type;
-        this.back_url = SCRYFALL_CARD_BACK_IMAGE_URL;
-        this.front_url = SCRYFALL_CARD_BACK_IMAGE_URL;
-        this.query = SCRYFALL_API_URL + '"' + name + '"';
+        this.back_url = DEFAULT_CARD_BACK_IMAGE_URL;
+        this.front_url = DEFAULT_CARD_BACK_IMAGE_URL;
         this.uri = "";
         this.tokens = [];
         this.failed = false;
         this.id = -1;
+        this.collectorNumber = "";
+        this.setCode = "";
```

Comment:
this sounds like a function name, perhaps change it to something like expansionCode

Reply:
I do agree. I can explain tough, it was extracted from Scryfall payload.
I'll change that to expansionCode, which is indeed better.